test/core/slice/slice_test.cc

```diff
@@ -395,14 +394,19 @@ size_t SumSlice(const Slice& slice) {
 TEST(SliceTest, ExternalAsOwned) {
   auto external_string = std::make_unique<std::string>(RandomString(1024));
   Slice slice = Slice::FromExternalString(*external_string);
-  const auto initial_sum = SumSlice(slice);
+  const size_t initial_sum = SumSlice(slice);
   Slice owned = slice.AsOwned();
   EXPECT_EQ(initial_sum, SumSlice(owned));
   external_string.reset();
   // In ASAN (where we can be sure that it'll crash), go ahead and read the
   // bytes we just deleted.
   if (BuiltUnderAsan()) {
-    ASSERT_DEATH({ gpr_log(GPR_DEBUG, "%" PRIdPTR, SumSlice(slice)); }, "");
+    ASSERT_DEATH(
+        {
+          size_t sum = SumSlice(slice);
+          VLOG(2) << sum;
+        },
```

Comment:
Using VLOG(2) << SumSlice(slice) does not work because the absl macro expands around this call, and then the ASSERT_DEATH does not work the way we want it to work.